pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -365,6 +365,10 @@ type NooBaaStatus struct {
 	// BeforeUpgradeDbImage is the db image used before last db upgrade
 	// +optional
 	BeforeUpgradeDbImage *string `json:"beforeUpgradeDbImage,omitempty"`
+
+	// PostgresMajorVersion is the major version of postgress db image
+	// +optional
+	PostgresMajorVersion string `json:"postgresMajorVersion,omitempty"`
```

Comment:
Please define an alias with const values instead of using string directly

Reply:
sure